editor/src/app/sites/site.component.ts

```diff
@@ -66,6 +67,8 @@ export class SiteComponent implements OnInit {
 
   @Output() inputFocus = new EventEmitter();
 
+  @Select('app') public current_site$: Observable<AppStateModel>;
```

Comment:
please use camel case as everywhere else - `currentSite$` instead of `current_site$`.